### ISSUE TITLE
Allow DEFAULT NULL column constraint

### ISSUE THREAD

**Issue by cwimberger** [alicebob/sqlittle#28]
I have the following column definition: `style_id STRING DEFAULT NULL`

I tried to modify the columnConstraint definition in `parser.go.y` to:

```
columnConstraint:
	PRIMARY KEY sortOrder autoincrement {
		$$ = ccPrimaryKey{$3, $4}
	} |
	UNIQUE {
		$$ = ccUnique(true)
	} |
	NULL {
		$$ = ccNull(true)
	} |
	NOT NULL {
		$$ = ccNull(false)
	} |
	COLLATE identifier {
		$$ = ccCollate($2)
	} |
	DEFAULT signedNumber {
		$$ = ccDefault($2)
	} |
	DEFAULT literal {
		$$ = ccDefault($2)
	} |
	DEFAULT NULL {
		$$ = ccDefault(nil)
	}
```

and added another column definition test case:

`"i0 not null default null":             &ColumnDef{Name: "i0", Default: nil},`

I can't figure out how to modify `makeColumnDef` in sql.go to allow the value. Would be great if you could help out here again!

**Comment by alicebob** [comment]
No problem. Does #29 work for you?

**Comment by cwimberger** [comment]
Yes, works like a charm! Thanks!

**Comment by alicebob** [comment]
Great! Merged and it's release v1.3.3
